docs/blog/newsletter-may-2022.rst

```diff
@@ -0,0 +1,118 @@
+
+Those in opposition to video say the main problems are the inaccessibility of video content: they're not searchable, so users cannot quickly browse the video to find what they need. Another issue is the ineffectiveness of video interface controls, which prevents users from adjusting the video to their liking, such as by slowing down the speed or pausing in exactly the right place. Plus there are the  business problems. Efforts to create videos may be considered too labor-intensive or may be disrupted if resources are limited, as videos require specialized skills that can be costly to employ.
+
+On the other side, proponents of video content say it can provide a lot of value - if done right. That is, videos should be brief, limit extensive hands-on engagement, apply accessibility best practices like captions and timestamps, and be professionally edited to establish credibility. Another way to ensure video content is well-received is to consider its optimal use. For example, videos are excellent when used as product demos but may not be ideal if used as the sole means of information in the product documentation.
+
+It's crucial to recognize when video content is valuable and when it is not; it comes back to the key goals of documentation, delivering value to users and facilitating understanding. None of the participants felt that video content could completely replace written content - but it can certainly be used to supplement it.
+
+-----------------------------------
+Templates for concept documentation
+-----------------------------------
+
+Most of us have written concept or explanatory documentation at some point. But unlike procedures, which are easier to formalize, concept docs can vary hugely in form - it all depends on the ideas you're trying to explain. So is it possible to have a template for concept docs?
+
+One community member suggested a rough outline:
+
+- Present the context and the `learning objective <https://www.sciencedirect.com/topics/social-sciences/learning-objective>`__ for the page.
+- Start with an overview of the concept and problem space - something like a Wikipedia-style `lead section <https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Lead_section>`__.
+- Break down the concept into smaller pieces, and explain those in turn.
+- Anchor each explanation with examples.
+- Link to action oriented content - what a reader can do with their new understanding.
+
+But it's hard to be much more prescriptive than this, because the form of an explanation article depends strongly on its scope. And figuring out the scope is a fundamental part of the challenge. It should be grounded in **who** is going to read it and **for what**. You'll need to draw the boundaries around the domain, and then explain both what's inside, but also the key relationships to the things outside.
```

Comment:
```suggestion
But it's hard to be much more prescriptive than this, because the form of an explanation article depends strongly on its scope. And figuring out the scope is a fundamental part of the challenge. It should be grounded in **who** is going to read it and **for what**. You'll need to draw the boundaries around the domain, and then explain both what's inside as well as the key relationships to the things outside.
```